src/Xunit.StaFact/Xunit.StaFact.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>net472;netstandard2.0;net6.0;net6.0-windows</TargetFrameworks>
+    <TargetFrameworks Condition="!$([MSBuild]::IsOsPlatform('Linux'))">$(TargetFrameworks);net6.0-macos10.15</TargetFrameworks>
```

Comment:
Failure to build with macOS workload on Linux is filed at https://github.com/xamarin/xamarin-macios/issues/17287